alembic/versions/9359297269a8_convert_ribbon_to_a_multichoice_column.py

```diff
@@ -0,0 +1,38 @@
+"""Convert ribbon to a multichoice column.
+
+Revision ID: 9359297269a8
+Revises: 167243c0e86c
+Create Date: 2017-07-21 09:18:38.444238
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '9359297269a8'
+down_revision = '167243c0e86c'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
+
+if is_sqlite:
+    op.get_context().connection.execute('PRAGMA foreign_keys=ON;')
+    utcnow_server_default = "(datetime('now', 'utc'))"
+else:
+    utcnow_server_default = "timezone('utc', current_timestamp)"
+
+
+def upgrade():
+    op.alter_column('attendee', 'ribbon', type_=sa.Unicode(), server_default = '', nullable = False)
+
+
+def downgrade():
+    op.alter_column('attendee', 'ribbon', type_=sa.Integer(), server_default=154973361, nullable=False)
```

Comment:
This downgrade is totally not gonna work if any attendees are already assigned multiple ribbons. It'll probably blow up being unable to parse `"1234,4567"` into an integer.

I don't think we really care about downgrades? But if we did, we'd want to select all the attendees that have a comma in their ribbon column, and selectively choose the right ribbon for them. Like department head takes priority over volunteer, etc...

Once that completes, then run the alter column statement.

But since we'll probably never run a downgrade migration, I'm happy merging this as-is.